### PR TITLE
Use Nuget for Data.Analysis

### PR DESCRIPTION
**Pull request by pgovind**
I tried this out on Binder and it works fine! I can't add reviewers for some reason, so tagging here:

@eerhardt @jonsequitur

### CODE REVIEW

File: NotebookExamples/csharp/Samples/HousingML.ipynb [hunk 1]
```diff
@@ -81,10 +81,9 @@
     }
    ],
    "source": [
-    "#r \"nuget:RestoreSources=https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json\"\n",
     "#r \"nuget:Microsoft.ML,1.4.0\"\n",
     "#r \"nuget:Microsoft.ML.AutoML,0.16.0\"\n",
-    "#r \"nuget:Microsoft.Data.Analysis,0.1.1-e191107-1\""
+    "#r \"nuget:Microsoft.Data.Analysis, 0.1.0\""
```
Comment (eerhardt) [inline NotebookExamples/csharp/Samples/HousingML.ipynb]: Do we need/like the space between the comma and the version?

**Comment by eerhardt** [comment]
We should also update the F# sample

https://github.com/dotnet/try/blob/master/NotebookExamples/fsharp/Samples/HousingML.ipynb

File: NotebookExamples/csharp/Samples/HousingML.ipynb (hunk 1 above)
Comment (pgovind) [reply NotebookExamples/csharp/Samples/HousingML.ipynb]: Fixed

**Comment by pgovind** [comment]
I _think_ the test failures are unrelated. Can somebody kick off a new build maybe?

**Comment by pgovind** [comment]
Thanks @jonsequitur

**Pull request merged**